src/Login.ts

```diff
@@ -123,8 +124,11 @@ export default class Login {
 
     public async getFlows(): Promise<Array<LoginFlow>> {
         const client = this.createTemporaryClient();
-        const { flows } = await client.loginFlows();
-        this.flows = flows;
+        const { flows }: { flows: LoginFlow[] } = await client.loginFlows();
+        // If an m.login.sso flow is present that is flagged as being for MSC3824 OIDC compatibility then we only return that flow
+        const oidcCompatibilityFlow =
+            flows.find(f => f.type === "m.login.sso" && f["org.matrix.msc3824.delegated_oidc_compatibility"]);
```

Comment:
Don't we have a const for `org.matrix.msc3824.delegated_oidc_compatibility`?

Reply:
Also it might want to be an UnstableValue (somehow) so it works with both the prefixed version and non-prefixed.

Reply:
I've added const values.

I've also added support for both stable and unstable prefixes. However, I thought this was against best practise?

FYI - I saw that react-sdk doesn't have `UnstableValue` unlike js-sdk.

Reply:
@hughns yes, its in the js-sdk so that you can import it in any layer above